### ISSUE TITLE
freereceivesms.com: detection

### ISSUE THREAD

**Issue by solxr** [uBlockOrigin/uAssets#11804]
### Prerequisites

- [X] I read and understood the [policy as to what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets)
- [X] I verified that the issue has not already been reported (use [this button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out)
- [X] I forced an update of my filter lists (**how to do this:** [click _"Purge all caches"_](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#purge-all-caches), then [click _"Update now"_](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#update-now))
- [X] I did not remove any of the [default filter lists](https://user-images.githubusercontent.com/585534/91296476-ab9d8c00-e76a-11ea-9fb9-87d6861bdf73.png), or I verified that the issue is not caused by removing any of the default lists
- [X] I did not enable external filter lists, or I verified that the issue still occurs without enabling external filter lists
- [X] I do not have custom filters/rules, or I verified that the issue still occurs without custom filters/rules
- [X] I am not using [another content blocker along uBO](https://twitter.com/gorhill/status/1033706103782170625) (also mind browser built-in blockers)
- [ ] I did not answer truthfully to **all** the above checkpoints

### URL address of the web page

`https://www.freereceivesms.com/`

### Category

detection

### Describe the issue

Adblock Detected.

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

![chrome_e948A5Mycw](https://user-images.githubusercontent.com/71159832/154784351-a91b68c6-1beb-4f96-962b-9cb4fca28b62.png)

</details>

### Configuration

<details>

```yaml
uBlock Origin: 1.41.2
Chromium: 98
filterset (summary): 
  network: 79893
  cosmetic: 96419
  scriptlet: 16474
  html: 0
listset (total-discarded, last updated): 
  removed: 
    spa-1: null
    ublock-abuse: null
    urlhaus-1: null
  added: 
    fanboy-annoyance: 60421-154, 6h.48m
  default: 
    user-filters: 0-0, never
    easylist: 63768-9, 6h.48m
    easyprivacy: 26576-122, 6h.48m
    plowe-0: 3670-803, 6h.48m
    spa-0: 3783-29, 6h.48m
    ublock-badware: 3990-76, 6h.48m
    ublock-filters: 30243-182, 6h.48m
    ublock-privacy: 203-10, 6h.48m
    ublock-unbreak: 1716-42, 6h.48m
filterset (user): [empty]
trustedset: 
  added: [array of 10 redacted]
modifiedUserSettings: [none]
modifiedHiddenSettings: [none]
supportStats: 
  allReadyAfter: 471 ms (selfie)
  maxAssetCacheWait: 264 ms
popupPanel: 
  blocked: 6
  blockedDetails: 
    freereceivesms.com: 1
    addthis.com: 1
    google.com: 2
    googlesyndication.com: 1
    googletagmanager.com: 1
```
</details>

**Comment by solxr** [comment]
> Try this filter:
> 
> ```
> freereceivesms.com##body>.modal-backdrop.fade.show,body>.modal.fade.show
> ```

Worked, thanks.

**Comment by solxr** [comment]
![chrome_7t7ozrIEbb](https://user-images.githubusercontent.com/71159832/154852310-fd18c2ca-3e3a-418d-aca1-203ecae5028f.png)

**Comment by solxr** [comment]
Sorry, fixed now?

**Comment by krystian3w** [comment]
#17965

**Comment by krystian3w** [comment]
update #10091 fix #11804
